README.md

```diff
@@ -4,99 +4,91 @@
 
-### Question #3
+ ### Question #3
 
 Convert the following ActiveRecord sequence to Mongoose:
 
-```rb
-@andy = Instructor.find_by(name: "Andy")
-@andy.wishlist_items.create(description: "Resin Laying Deer Figurine, Gold")
-```
+Instructor.findOne({name: "Andy"}, function(err, andy){
+  andy.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
+  andy.save();
+};
+});
+ ```
 
-```js
-// Your answer...
-```
+ ### Question #4
+Convert the following create method in Mongoose to ActiveRecord.
+ }
+ ```
 
-### Question #4
+-```rb
+```@author = Author.create!(name: params[:name])
 
+ ```
+ ## Express
 Convert the following create method in Mongoose to ActiveRecord.
 
-```js
-  var authors = {
-  create: function(req, res){
-  var author = new AuthorModel({name: req.body.name})
-  author.save(function(err){
-    if (!err){
-      res.redirect("authors")
-    }
-  })
-  }  
-}
-```
-
-```rb
+ ### Question #5
+ How does module.exports help us with separation of concerns?
 
-```
-## Express
 
-### Question #5
+module.exports is the thing (sometimes an object) that is returned as the result of a require call.
 
-How does module.exports help us with separation of concerns?
 
-```text
+###Question #6
+Write one Express route for each of the four HTTP methods.
+ Then, make each route respond with a one-word string containing the RESTful action that would most likley be associated with this route.
+ var express = require("express");
+ var app = express();
 
-```
 
-### Question #6
+1 app.get("/", function(req, res){
+res.send("INDEX");
+});
 
-Write one Express route for each of the four HTTP methods.
+2  app.put("/:id", function(req, res){
+  res.send("UPDATE");
+});
 
-Then, make each route respond with a one-word string containing the RESTful action that would most likely be associated with this route.
+3  app.post("/", function(req, res){
+  res.send("CREATE");
+});
 
-```js
-var express = require("express");
-var app = express();
+4  app.delete("/:id", function(req, res){
+  res.send("DELETE");
+});..
 
-// Your code starts here...
+ ```
 
-```
+var app = express();
 
-```js
-// Your answer...
-```
 ### Question #7
 
-Describe the differences between Express and Rails as backend frameworks.
+ Describe the differences between Express and Rails as backend frameworks.
 
-```text
+Express is a framework that leaves a lot of the specifics and decisions about the structure to the user/developer, whereas Rails is specific and much be structured a certain way in order for the framework to function.
 
-```
+ ```
 
-### Question #8
+ ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? 
+-What is the importance of using body-parser in our express application for post requests?
+What is the importance of using body-parser in our express application for post requests?
 
-```js
+We use body-parser in express and node applications to take the data from users and make post requests.
```

Comment:
We need body parser middleware in order to process user input received through a form